projects/plugins/vaultpress/license.txt

```diff
@@ -0,0 +1,357 @@
+This program is free software; you can redistribute it and/or modify
```

Comment:
Should be able to get rid of this here. Since composer.json declares the license as "GPL-2.0-or-later", the mirroring build script will copy it in from .github/licenses/GPL-2.0-or-later.txt.

Reply:
Done.